lib/Onyx.js

```diff
@@ -1082,15 +1082,19 @@ function multiSet(data) {
 function applyMerge(existingValue, changes) {
     const lastChange = _.last(changes);
 
-    if (_.isArray(existingValue) || _.isArray(lastChange)) {
```

Comment:
I think we are missing an edge case where it's possible to:

- Have an array
- Call merge with `undefined` (or the last change is undefined)
- We will set the value to that

Can we ignore all `undefined` by filtering them out first?